text/src/autogluon/text/automm/optimization/utils.py

```diff
@@ -23,6 +23,7 @@
 def get_loss_func(
         problem_type: str,
         mixup_active: bool,
+        bce_type: bool,
```

Comment:
How about to change `bce_type` to `loss_function` and use None by default. The loss function can then be `bce` and will be used if it's regression problem.

Also, we need to revise the docstring below (add mixup_active and loss_function to parameters).